service/src/main/java/com/epam/esm/mapper/ServiceOrderMapper.java

```diff
@@ -19,7 +21,13 @@ public Order toEntity(OrderDto orderDto) {
     }
 
     public OrderDto toDto(Order order) {
-        return modelMapper.map(order, OrderDto.class);
+        OrderDto orderDto = modelMapper.map(order, OrderDto.class);
+        if (order.getCertificates() != null) {
```

Comment:
Please try to find the most relevant method from [CollectionUtils](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html) class.